src/controllers/album.js

```diff
@@ -0,0 +1,58 @@
+const db = require('../services/db');
+const getDb = require('../services/db');
+
+exports.create = async (req, res) => {
+  const db = await getDb();
+  const { name, genre, year } = req.body;
+  const artistId = req.params.artistId;
+
+  try {
+    await db.query(`INSERT INTO Album (name, genre, year, artistId) VALUES(?, ?, ?, ?)`, [
+        name, 
+        genre, 
+        year, 
+        artistId
+    ]);
+
+    res.sendStatus(201);
+  } catch (err) {
+    res.sendStatus(500).json(err);
+  }
+
+  db.close();
+};
+
+exports.read = async(req, res) => {
```

Comment:
From here to the end of the file the indentation needs fixing. Since you have prettier as part of the project you could format everything in no time at all, in VSCode all you need to do is right click somewhere in the file and click `format document`. You can also configure it so it will format the code automatically whenever you save a file (recommended approach)